concrete/src/Localization/Locale/Service.php

```diff
@@ -99,6 +99,9 @@ public function addHomePage(Locale $locale, Template $template, $name, $url_slug
         ]);
         $home->rescanCollectionPath();
 
+        // Copy the permissions from the canonical home page to this home page.
+        $home->acquirePagePermissions(1);
```

Comment:
I know we used to have the `HOME_CID` constant, do we still have something like that?